service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/ServiceClient.java

```diff
@@ -134,20 +139,16 @@ public void send(String deviceId, Message message) throws IOException, IotHubExc
      */
     public void send(String deviceId, String moduleId, Message message) throws IOException, IotHubException
     {
-        // Codes_SRS_SERVICE_SDK_JAVA_SERVICECLIENT_28_001: [The function shall throw IOException if the member AMQP sender object has not been initialized]
-        if (this.amqpMessageSender == null)
-        {
-            throw new IOException("AMQP sender is not initialized");
-        }
-        // Codes_SRS_SERVICE_SDK_JAVA_SERVICECLIENT_28_002: [The function shall call send() on the member AMQP sender object with the given parameters]
-        this.amqpMessageSender.send(deviceId, moduleId, message);
+        AmqpSend amqpMessageSender = new AmqpSend(hostName, userName, iotHubConnectionString, this.iotHubServiceClientProtocol, this.sasTokenExpiryTime);
+        amqpMessageSender.send(deviceId, moduleId, message);
     }
 
     /**
-     * Provide asynchronous access to open()
-     *
+     * Provide asynchronous access to open(), which does nothing anymore
+     * @deprecated This method no longer does anything. See {@link #open()} documentation for more details
```

Comment:
q - why did we have an async open() to begin with?

Reply:
All of the service client operations had an async flavor, but I don't know why.